react/hooks/useCumulativeHeightState.tsx

```diff
@@ -59,7 +59,7 @@ const useCumulativeHeightState = () => {
   const getAccumulatedHeight = (index: number) => {
     const sortedIndices = Object.keys(state)
       .map(key => parseInt(key, 10))
-      .sort()
+      .sort((a, b) => a - b)
```

Comment:
Probably no one will have 9+ sticky rows, but if they do, this would sort it alphabetically `1, 10, 2, 3, 4...`.